administrator/components/com_cookiemanager/tmpl/cookies/default.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cookiemanager
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+\defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Associations;
+use Joomla\CMS\Language\Multilanguage;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Layout\LayoutHelper;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Session\Session;
+
+HTMLHelper::_('behavior.multiselect');
+
+$user      = Factory::getUser();
```

Comment:
` Factory::getUser();` is deprecated - always use `Factory::getApplication()->getIdentity()` instead
```suggestion
$user      = Factory::getApplication()->getIdentity();
```